src/clients/update_proposal.rb

```diff
@@ -187,8 +187,15 @@ def main
 
         products = Y2Packager::Resolvable.find(kind: :product)
         # stores the proposal text output
-        @summary_text = Packages.product_update_summary(products)
+        summary_text = Packages.product_update_summary(products)
           .map { |item| "<li>#{item}</li>" }.join
+        product_info = "<li>" + format(
+          _("Updating %{product} on root %{partition}"),
+          product:   RootPart.GetInfoOfSelected(:name),
+          partition: RootPart.selectedRootPartition
```

Comment:
Could you please check an installation with LVM? Just to be sure that `#selectedRootPartition` contains theh value in that case.

Reply:
yeap, here is partition selection and also summary. So we are consistent:
![show_partition_lvm](https://github.com/yast/yast-update/assets/478871/8ff1f637-e682-4032-a343-ac097ae529e3)
![upgrade_selection](https://github.com/yast/yast-update/assets/478871/6198a91f-93df-4afb-907b-e997a71d3c9f)